src/test/suite/fixtures/recorded/lineNumbers/takeRowFour.yml

```diff
@@ -5,7 +5,7 @@ command:
   partialTargets:
     - type: primitive
       selectionType: line
-      modifier:
```

Comment:
shall we add a test case that uses it like a proper mark?  Eg "take funk row five"

Reply:
We don't really have that for the other marks. There is no "take funk that" or similar. But it is treated like the other marks in the code so there really is no difference. If we should add more test for marks in general is a different question.

Reply:
I think I'd like to have that kind of test, but if you think it's better to do those in one pass that's fine too; we don't need to block this PR on that

Reply:
Exactly I think that we probably should do proper tests for all the marks as a separate thing and not just this one specifically.